src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -130,6 +130,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				scope: ConfigurationScope.APPLICATION,
 				tags: ['usesOnlineServices']
 			},
+			'extensions.autoUpdateDisabledExtensions': {
+				type: 'boolean',
+				description: localize('autoUpdateDisabledExtensions', "When enabled, automatically installs updates for disabled extensions. The updates are fetched from a Microsoft online service."),
+				default: false,
```

Comment:
I think disabling this option by default might be confusing to the users - some extensions are auto udpated by default and some are not.  So I would recommend to enable it by default so out of the box user has all extensions are auto updated by default and they can configure manually.

I would also change the description mentioning what happens when disabled - when disabled user disabled extensions are not auto updated.